webapp/javascript/participants.js

```diff
@@ -0,0 +1,40 @@
+/** Some Hard-coded data to display as participants temporarily. */
+var participant1 = {
```

Comment:
this file appears to conflict with the changes in Asha's pull request https://github.com/googleinterns/step121-2020/pull/21/. Asha's changes appear to be later than these, so I have approved that request. This request will need to be rebased with respect to Asha's changes once commited.

Working on the same files can often cause problems like this unfortunately, so good communication on who's doing what and why is key.